### ISSUE TITLE
Error in Homeassistant since update 2024.01 Deprecation of STATE_CLASS_MEASUREMENT and STATE_CLASS_TOTAL_INCREASING

### ISSUE THREAD

**Issue by pschyrembel258** [syssi/homeassistant-goecharger-mqtt#127]
Logger: homeassistant.components.sensor
Source: helpers/deprecation.py:204
Integration: Sensor (documentation, issues)
First occurred: 14:07:14 (4 occurrences)
Last logged: 14:07:14

STATE_CLASS_MEASUREMENT was used from goecharger_mqtt, this is a deprecated constant which will be removed in HA Core 2025.1. Use SensorStateClass.MEASUREMENT instead, please create a bug report at https://github.com/syssi/homeassistant-goecharger-mqtt/issues
STATE_CLASS_TOTAL_INCREASING was used from goecharger_mqtt, this is a deprecated constant which will be removed in HA Core 2025.1. Use SensorStateClass.TOTAL_INCREASING instead, please create a bug report at https://github.com/syssi/homeassistant-goecharger-mqtt/issues

**Comment by LinusHoppe** [comment]
I can add some more:
```
Logger: homeassistant.const
Source: helpers/deprecation.py:205
First occurred: 08:54:33 (12 occurrences)
Last logged: 08:54:33

POWER_KILO_WATT was used from goecharger_mqtt, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfPower.KILO_WATT instead, please create a bug report at https://github.com/syssi/homeassistant-goecharger-mqtt/issues
POWER_WATT was used from goecharger_mqtt, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfPower.WATT instead, please create a bug report at https://github.com/syssi/homeassistant-goecharger-mqtt/issues
TEMP_CELSIUS was used from goecharger_mqtt, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTemperature.CELSIUS instead, please create a bug report at https://github.com/syssi/homeassistant-goecharger-mqtt/issues
TIME_MILLISECONDS was used from goecharger_mqtt, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTime.MILLISECONDS instead, please create a bug report at https://github.com/syssi/homeassistant-goecharger-mqtt/issues
TIME_SECONDS was used from goecharger_mqtt, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTime.SECONDS instead, please create a bug report at https://github.com/syssi/homeassistant-goecharger-mqtt/issues
```

**Comment by arboeh** [comment]
Yep, can confirm this warning.

**Comment by syssi** [comment]
Fixed by version 0.22.0.
